volatility/framework/plugins/windows/cachedump.py

```diff
@@ -0,0 +1,131 @@
+
+from volatility.framework import interfaces, renderers
+from volatility.framework.configuration import requirements
+from volatility.framework.renderers import format_hints
+from volatility.framework.layers import intel
+from volatility.plugins.windows.registry import hivelist
+from volatility.plugins.windows import hashdump, lsadump, poolscanner
+from Crypto.Hash import HMAC
+from Crypto.Cipher import ARC4, AES
+from struct import unpack
+
+
+class Cachedump(interfaces.plugins.PluginInterface):
+    """Dumps lsa secrets from memory"""
+    @classmethod
+    def get_requirements(cls):
+        return [requirements.TranslationLayerRequirement(name = 'primary',
+                                                     description = 'Memory layer for the kernel',
+                                                     architectures = ["Intel32", "Intel64"]),
+            requirements.SymbolTableRequirement(name = "nt_symbols",
+                                                description = "Windows kernel symbols"),
+            requirements.PluginRequirement(name = 'hivelist', plugin = hivelist.HiveList, version = (1, 0, 0))
+            ]
+    @classmethod
```

Comment:
The formatting suggests yapf hasn't been run over this.  Ideally, this would have yapf run on it so that it fits the standard (but it's not a show stopper, just a nice to have).  5:)